src/python_testing/TC_OpstateCommon.py

```diff
@@ -1024,63 +1043,77 @@ async def TEST_TC_OPSTATE_BASE_2_5(self, endpoint=1):
         # STEP 1: Commission DUT to TH (can be skipped if done in a preceding test)
         self.step(1)
 
-        # STEP 2: Set up a subscription to the OperationCompletion event
-        self.step(2)
-        if self.pics_guard(self.check_pics(f"{self.test_info.pics_code}.S.E01")):
+        # Store PICS value for the Op Complete Event
+        opcomplete_pics = self.pics_guard(self.check_pics(f"{self.test_info.pics_code}.S.E01"))
```

Comment:
pics_guard is the thing that does the skip step - I think you'll get weird behaviour here if this is false. You can just use the check_pics function.

Reply:
Fixed